csharp/src/Google.Protobuf/JsonFormatter.cs

```diff
@@ -170,6 +183,27 @@ public void Format(IMessage message, TextWriter writer)
         }
 
         /// <summary>
+        /// Formats the specified field of the message as JSON.
+        /// </summary>
+        /// <param name="message">The message to format.</param>
+        /// <param name="field">The field of the message to format.</param>
+        /// <param name="writer">The TextWriter to write the formatted message to.</param>
+        /// <returns>The formatted field.</returns>
+        public void Format(IMessage message, FieldDescriptor field, TextWriter writer)
```

Comment:
I didn't consider the method name before, but how about `FormatField` or even `FormatFieldValue`? That conveys quite a lot of information in very little text.